packages/components/src/interfaces/complex/KeyStatistics.ts

```diff
@@ -28,6 +28,13 @@ export const KeyStatisticsSchema = Type.Object(
         maxItems: 4,
       },
     ),
+    url: Type.Optional(
+      Type.String({
+        title: "Link destination",
+        description: "When this is clicked, open:",
+        format: "link",
```

Comment:
note: the `link` format here implies that this can accept **any one of**: `internal link/ext link/image/file`. 

from what this component sounds liek it's meant to be used for, we should probably dissuade the last 2? but it might be very annoying to fix tbh - will let sehyun make the call. imo, opening files/images are fine in my book becauase they could convey more meaningful info (ie, this `url` is a see more for the key stats) and is also a pattern that we find elsewhere (eg: see more on current financials downloads a pdf report for companies)

Reply:
Oh Isn't it `mailto` instead of `image`? 

`Internal link` is what MOH needs now so by right we could start with just that and expand later, but then we'd need to change the link selector on studio right. Doesn't seem like worth it

Technically there isn't a huge risk to supporting all 4 imo, like you said you could link a PDF report or email someone about the stats

If it is indeed `image` instead of `mailto` err that one I'd be a bit worried, strange to link an image to a block

Reply:
> note: the `link` format here implies that this can accept **any one of**: `internal link/ext link/image/file`. 

Yup! Noted in the PR that it was intentional (for now) since there're quite a number of pending PR and changes made to this link component so I don't want to edit the component to have the disabled option as it will have many conflicting changes anyway

Reply:
update - we can use the `linkType` prop in my PR to do this

Reply:
re: https://github.com/opengovsg/isomer/pull/782